docs/source/tutorials/projects.rst

```diff
@@ -0,0 +1,1213 @@
+You will need an API key with the``label_logs.read`` permission.
+The ``get_label_logs`` takes a number of optional parameters to narrow down the retrieved logs:
+
+.. code-block::
+
+    from encord.client import EncordClient
+
+    client = EncordClient.initialise("<project_id>", "<project_api_key>")
+    logs = client.get_label_logs(user_hash=<user_hash>)
+    for log in logs:
+        print(log)
+
+.. code-block::
+
+    def get_label_logs(
+        self,
+        user_hash: str = None,
+        data_hash: str = None,
+        from_unix_seconds: int = None,
+        to_unix_seconds: int = None
+        )
+        -> List[LabelLog]
+
+
+Models
+======
+
+.. note::
+    This section is undergoing a reconstruction [Tuesday, May 10th 2022].
+
+The |sdk| allows you to interact with the Encord model features.
+Our model library includes state-of-the-art classification, object detection, segmentation, and pose estimation models.
+
+Creating a model row
+--------------------
+
+The easiest way to get started with creating a model row is to navigate to the 'models' tab in your project. Create a model and set parameters accordingly.
+
+.. figure:: /images/python_sdk_model_create.png
+
+    Getting model API details.
+
+Click on the 'Model API details' button to toggle a code snippet with create model row API details when you are happy with your selected parameters.
+
+.. code-block::
+
+    from encord.client import EncordClient
+    from encord.constants.model import *
+
+    # Initialize project client
+    client = EncordClient.initialise(
+      "<project_id>",  # Project ID
+      "<project_api_key>"  # API key
+    )
+
+    model_row_uid = client.create_model_row(
+                                title='Sample title',
+                                description='Sample description',  # Optional
+                                features=['feature_uid_1', 'feature_uid_2', ...], #  List of feature feature uid's (hashes) to be included in the model.
+                                model=FASTER_RCNN
+                            )
+    print(model_row_uid)
+
+
+The following models are available, and are all imported using ``from encord.constants.model import *``.
+
+.. code-block::
+
+    # Classification
+    FAST_AI = "fast_ai"
+    RESNET18 = "resnet18"
+    RESNET34 = "resnet34"
+    RESNET50 = "resnet50"
+    RESNET101 = "resnet101"
+    RESNET152 = "resnet152"
+    VGG16 = "vgg16"
+    VGG19 = "vgg19"
+
+    # Object detection
+    YOLOV5 = "yolov5"
+    FASTER_RCNN = "faster_rcnn"
+
+    # Instance segmentation
+    MASK_RCNN = "mask_rcnn"
+
+
+Training
+--------
+
+Navigate to the 'models' tab in your project to get started with model training.
+Start by creating a model row using the Python SDK or by following the :xref:`create_model_guidelines`.
+You can also use an existing model by clicking on the 'train' button.
```

Comment:
To match the web app docs (seeing as there are web app screenshots here) probably should have the train button in italic rather than in quotes